### PR TITLE
Add example that uses sensors

### PR DESCRIPTION
**Pull request by dbrgn**
Can be run with `cargo run --example with_sensors`

### CODE REVIEW

File: examples/with_sensors.rs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+
+fn main() {
+    env_logger::init().unwrap();
+
+    // Create new minimal Status instance
+    let status = api::Status::new(
+        "coredump",
+        "https://www.coredump.ch/logo.png",
+        "https://www.coredump.ch/",
```
Comment (rnestler) [inline examples/with_sensors.rs]: Shouldn't we better use something like www.example.com?

Comment (dbrgn) [inline examples/with_sensors.rs]: I thought some self-advertising can't hurt?

Comment (rnestler) [inline examples/with_sensors.rs]: True :wink:

**Comment by rnestler** [comment]
:+1:

**Pull request merged**